app/src/main/java/sdp/moneyrun/ui/map/MapActivity.java

```diff
@@ -434,7 +440,7 @@ private void setupChronometer() {
             game.setStartTime(current, false);
         } else {
             long start = game.getStartTime();
-            chronometerCounter = (int) (current - start + 2);
+            chronometerCounter = (int) (current - start + 6);
```

Comment:
What is the +6?

Reply:
There is about a 6 second difference between when the time is fetched and when it is displayed